turf/src/commonMain/kotlin/org/maplibre/spatialk/turf/measurement/bbox.kt

```diff
@@ -84,7 +84,8 @@ public fun bbox(geometry: MultiPolygon): BoundingBox = computeBbox(geometry.coor
  * @return A [BoundingBox] that covers the geometry.
  */
 @ExperimentalTurfApi
-public fun bbox(feature: Feature): BoundingBox = computeBbox(feature.coordAll() ?: emptyList())
+public fun bbox(feature: Feature<Geometry>): BoundingBox =
```

Comment:
I think because we have `Feature<out T : Geometry>`, we can write `Feature<*>` most (all?) places we'd otherwise write `Feature<Geometry>`